docs/src/main/tut/docs/index.md

```diff
@@ -177,14 +175,13 @@ consumer.assign(topicName, initialOffsets).unsafeRunSync
 And we can now read a stream of records from our Kafka topic:
 
 ```tut
-val messages = consumer.rawRecordStream(1.second).take(5).compile.toVector.unsafeRunSync
+val messages = consumer.recordStream(1.second).take(5).compile.toVector.unsafeRunSync
 ```
 
 To clean up after ourselves, we'll close and shut down our resources:
 ```tut
 consumer.close.unsafeRunSync
 producer.close.unsafeRunSync
-blockingContext.shutdown
```

Comment:
I think we still need the explicit shutdown, since `ConsumerApi.Avro4s.create` ultimately calls 
```
  def apply[F[_]: Async: ContextShift, K, V](
      configs: (String, AnyRef)*
  ): F[ConsumerApi[F, K, V]] =
    for {
      c <- createKafkaConsumer[F, K, V](configs: _*).map(ConsumerImpl(_))
      e <- BlockingContext.default
    } yield ShiftingConsumerImpl(c, e)
```
so the blocking context is created, but not shut down.

Reply:
Yeah I have a feeling this is why the travis build is hanging on the `tut` part now.

Reply:
Refactoring to use resources would probably require additional corresponding changes to the prose, but that's another option.

Reply:
I think I actually messed things up, where most of these `resource`/`stream` functions are not closing the execution context. Working on a commit to fix all of this.

Almost wondering if we should only provide smart constructors that return `Resource`/`Stream` and not the raw `F[ConsumerApi[F, K, V]]`.

Reply:
If we do keep things like `create` then I think they have to return `F[(ConsumerApi[F, K, V], ExecutionContextExecutorService)]`.